release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2542,3 +2542,48 @@ With this update, users can now collect the `virt_platform` metric. The `virt_pl
+
+* Before this update, the Pipeline ServiceAccount did not use secrets created during the git import flow for private git repositories, causing these Pipelines to fail. This update fixes the problem by adding annotations to the secrets and to the Pipeline ServiceAccount. Pipelines for private git repositories now run correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[*BZ#1970485*])
+
+* Previously, health checkers for Google Cloud Load Balancer left stale conntrack entries on cluster hosts, and this caused network interruptions to API server traffic that used the Google Cloud Load Balancer. This fix prevents the health check traffic from looping through the hosts, and this removed the network disruptions to the API server. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949348[*BZ#1949348*])
+
+* Previously, in the web console's Topology view, visualizations of Knative services displayed build status and repository information in the wrong location. This update adjusts which data are passed to the UI, and now only the correct information is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954962[*BZ#1954962*])
+
+* Previously, some `CredentialsRequest` custom resources with predefined Google Cloud Platform permissions were rejected when they were applied to a GCP project that did not use all the predefined permissions. With this fix, the Cloud Credentials Operator periodically fetches a list of allowed permissions to test, so now `CredentialsRequest` custom resources do not fail permissions checking. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1958983[*BZ#1958983*])
+
+* The fix for https://bugzilla.redhat.com/show_bug.cgi?id=1953097[*BZ#1953097*] enabled the CoreDNS bufsize plugin with a buffer size of 1232 bytes, even though some DNS resolvers are capable of handling a maximum of only 512 bytes. As a result, some DNS resolvers were unable to receive messages from the DNS Operator. This fix sets the buffer size to 512 bytes so those DNS resolvers can receive messages as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1967766[*BZ#1967766*])
+
+* Previously, root privileges were required to set extended file attributes while extracting RHEL images, which caused `oc image extract` to fail for normal users. With this update, `oc` checks user permissions and sets extended attributes only for the root user. `oc image extract` works correctly for both root and normal users. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969928[*BZ#1969928*])
```

Comment:
s/`oc image extract`/the `oc image extract` command/

^ x2

s/`oc`/the OpenShift CLI (`oc`)/

s/RHEL/Red Hat Enterprise Linux (RHEL)/

Reply:
Agreed! Changed!